config/canonical.json

```diff
@@ -14224,6 +14227,7 @@
       "brand": "Cheesecake Factory",
       "brand:wikidata": "Q1045842",
       "brand:wikipedia": "en:The Cheesecake Factory",
+      "cuisine": "dessert",
```

Comment:
Hmm, I'd probably call this one `american` instead of `dessert`.. They serve all kinds of stuff there.